web/app/models.py

```diff
@@ -167,5 +167,17 @@ def to_dict(self):
             "rated": self.rated,
         }
 
+
+class Tournament(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    event_name = db.Column(db.String(80))
+    start_date = db.Column(db.DateTime())
+    venue = db.Column(db.String(80))
+    director = db.Column(db.String(80))
+    pairing = db.Column(db.String(80))
+    rule_set = db.Column(db.String(80))
```

Comment:
There should be a field that's whether or not the tournament has been 'submitted' to the ratings database. Once a tournament is submitted, it shouldn't be delete-able.